iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/IcebergTableUtil.java

```diff
@@ -258,4 +259,28 @@ public static void cherryPick(Table table, long snapshotId) {
     LOG.debug("Cherry-Picking {} to {}", snapshotId, table.name());
     table.manageSnapshots().cherrypick(snapshotId).commit();
   }
+
+  public static TransformSpec.TransformType getTransformType(Transform<?, ?> transform) {
+    String transformString = transform.toString();
+    if (transformString.contains("bucket")) {
```

Comment:
introduce fromString in TransformSpec.TransformType

Reply:
Done.